llvm/lib/Transforms/InstCombine/InstCombineCompares.cpp

```diff
@@ -3721,10 +3728,6 @@ Instruction *InstCombinerImpl::foldICmpBinOpWithConstant(ICmpInst &Cmp,
     if (Instruction *I = foldICmpSubConstant(Cmp, BO, C))
       return I;
     break;
```

Comment:
```suggestion
    break;
    case Instruction::Or:
    if (Instruction *I = foldICmpOrConstant(Cmp, BO, C))
      return I;
    [[fallthrough]];
```
I prefer the switch+fallthrough approach.